airflow/providers/sftp/CHANGELOG.rst

```diff
@@ -24,6 +24,18 @@
 Changelog
 ---------
 
+4.2.1
+.....
+
+Misc
+~~~~
+
+* ``Update codespell and fix typos (#28568)``
+* ``[misc] Get rid of 'pass' statement in conditions (#27775)``
```

Comment:
We don't typically add typo commits to release notes, and not sure it's worth releasing this set of changes.

Reply:
same comment as above :)

Reply:
Same comment as above. If there are code changes -> release. Early and often. This is the mantra I will keep on repeating, becaus I think avoiding releases is optimising the wrong thing. 

The whole automation about releasing providers was built around that premise and mantra.